app/controllers/claims/schools/claims/mentors_controller.rb

```diff
@@ -8,13 +8,17 @@ def new; end
 
   def create
     if claim_mentors_form.save
-      redirect_to(
-        edit_claims_school_claim_mentor_training_path(
-          @school,
-          claim,
-          claim.mentor_trainings.without_hours.first,
-        ),
-      )
+      if claim.mentor_trainings.without_hours.any?
```

Comment:
This probably should be a method in the form object but considering the situation is fine.

Reply:
Agreed. I'll clean this up as soon as I can. Thanks!